src/genpy/rostime.py

```diff
@@ -395,6 +396,11 @@ def __floordiv__(self, val):
         """
         t = type(val)
         if t in (int, long):
+            warnings.warn(
+                'The Duration.__floordiv__(integer) function is ill-defined. '
+                'The floor operation is applied independently on the seconds as well as the nanoseconds. '
+                'For true division use the operator `/` (which requires `from __future__ import division` in Python 2) or `operator.truediv` instead.',
```

Comment:
Nvm. Will updated the PR with another line added to the warning: https://github.com/ros/genpy/pull/58/files#diff-305939312869835e69b57abda55dd404R402